heartbeat/nfsnotify.in

```diff
@@ -204,7 +204,14 @@ copy_statd()
 	cp -rpfn $src/sm $src/sm.bak $src/state $dest > /dev/null 2>&1
 
 	# make sure folder ownership and selinux lables stay consistent
-	[ -n "`id -u rpcuser`" -a "`id -g rpcuser`" ] && chown rpcuser.rpcuser "$dest"
+	# When using nfsnotify resources on the debian system, the statd user replaces the rpcuser user 
+        local rpcuser_exist=`grep rpcuser /etc/passwd` 
```

Comment:
You should follow the indentation of the rest of the code here (seems to be tabs).

Can you also update it to not use -a for the part of the code you're modifying?
https://wiki.bash-hackers.org/commands/classictest#why_you_should_avoid_using_-a_and_-o